<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleCollectorUi</class>
 <widget class="QMainWindow" name="SampleCollectorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="lbPatientIdDesc">
          <property name="text">
           <string>User ID:</string>
          </property>
          <property name="buddy">
           <cstring>cbPatientId</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KHistoryComboBox" name="cbPatientId">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbPatientNameDesc">
          <property name="text">
           <string comment="Name of the user">Name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lbPatientName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pbSelectPatient">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="Select user associated with the entered id (fetch)">Select</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbSearchPatient">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string comment="Search for user">Search</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pbDetails">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Details</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pbDelete">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCommandLinkButton" name="pbRepeat">
         <property name="text">
          <string>Repeat</string>
         </property>
         <property name="description">
          <string>The interviewer reads out words on the screen which are then repeated and recorded by the patient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="pbTraining">
         <property name="text">
          <string>Training</string>
         </property>
         <property name="description">
          <string>The user reads training texts from the screen.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCommandLinkButton" name="pbInterview">
         <property name="text">
          <string>Interview</string>
         </property>
         <property name="description">
          <string>The user answers interview questions freely. The answers are recorded.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>494</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string comment="Quit the application">Quit</string>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="actionTraining">
   <property name="text">
    <string>Training</string>
   </property>
  </action>
  <action name="actionInterview">
   <property name="text">
    <string>Interview</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionInstitutions">
   <property name="text">
    <string>Institutions</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KHistoryComboBox</class>
   <extends>QComboBox</extends>
   <header>khistorycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>SampleCollectorUi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
